pkg/manager/librarian/librarian.go

```diff
@@ -203,10 +237,16 @@ func (m *Manager) CatalogForRedundancyGroup(ctx context.Context, kube kclient.Cl
 		a := &Allocator[uint16]{
 			Values: NewNextFreeValueFromVLANRanges(m.cfg.VPCIRBVLANRanges),
 		}
+		extVlans, err := a.Allocate(cat.Spec.IRBVLANs, externals)
```

Comment:
Each call of allocate would free up the "unused" entries so we should only call it once.